### DISCUSSION TITLE
How in  ListRecords to move  edit/remove links to the left side?

### DISCUSSION THREAD

**Discussion by PetroGromovo** [filamentphp/filament#6374] [Help] [answered]
Hello,
Using  list of resource based on ListRecords edit/remove links are at right side and that is not good with 
multi columns list as I have to scroll to the right side for these links.

Can I move  edit/remove links to the left side?

Thanks in advance!

**Comment by danharrin** [comment, marked as answer]
`->actions([...], position: Tables\Actions\Position::BeforeColumns)`

**Reply by jinbatsu** [reply]
Just a little correction.. at least for me (at v3.2.121).
It should be:
```php
->actions([...], position: Tables\Enums\ActionsPosition::BeforeColumns)
```
